invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/macros/request_header.html

```diff
@@ -29,6 +29,7 @@
   </div>
 
   <h2 class="ui header">{{ request.title }}</h2>
+  <h4 class="ui header">{{ request.description|safe }}</h4>
```

Comment:
```suggestion
  <p class="ui header">{{ request.description|safe }}</p>
```

minor: can you also change it below, for the other macro?

Reply:
done